### PR TITLE
Replace Google Analytics code with Plausible

### PR DESCRIPTION
**Pull request by tomodwyer**
So that we can track usage in a privacy-friendly way.

### CODE REVIEW

**Review by madwort — APPROVED** [review]

File: euctr/templates/_base.html [hunk 1]
```diff
@@ -28,18 +28,7 @@
       {% include "_load_js.html" %}
     {% endif %}
 
-    {% if GOOGLE_TRACKING_ID %}
```
Comment (madwort) [inline euctr/templates/_base.html:31]: should we remove the other references to `GOOGLE_TRACKING_ID` while we're here?

Comment (madwort) [inline euctr/templates/_base.html:31]: should we remove the other references to GOOGLE_TRACKING_ID while we're here?

**Review by madwort — CHANGES_REQUESTED** [review]

File: euctr/templates/_base.html (hunk 1 above)
Comment (tomodwyer) [reply euctr/templates/_base.html:31]: Yes, good point - I'll add a commit now.

Comment (tomodwyer) [reply euctr/templates/_base.html:31]: @madwort updated, I assume I'm ok to remove the `euctr/frontend/context_processors.py` file as it only had references to `google_tracking_id` in it...?

Comment (madwort) [reply euctr/templates/_base.html:31]: looks ok to me.

**Review by madwort — APPROVED** [review]

**Comment by madwort** [comment]
Deployed & verified.

**Pull request merged**